### ISSUE TITLE
Editable installs unusable on first install

### ISSUE THREAD

**Issue by GerevSec** [python-poetry/poetry#5298]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- Windows 11 WSL Ubuntu 20.04
- 1.1.13
- [pyproject.toml](https://gist.github.com/GerevSec/c7d5ccf8590ac892b3229d0e30ba5440)

## Issue
Hey guys,

I've encountered a quite annoying issue while trying to build our python monorepo.
When using "edtiable" installs, poetry instructs you to first **add** the local package, and then edit the pyproject.toml to add "develop = true", **lock** it again and **install** it again.

However, since the first time you add a package it's automatically installed, the current virtual env is "stuck" with non-editable installation of the package.
In order to start seeing the effects, we would need to exit the venv, delete it and then create a new one that installs the local package as editable.

Any thoughts of perhaps making editable install a command-line argument for **poetry add**?

Cheers,
Ayal

**Comment by finswimmer** [comment]
Hello @GerevSec,

> Any thoughts of perhaps making editable install a command-line argument for poetry add?

The current preview release (1.2.0a2) have the `--editable` argument for it.

fin swimmer

**Comment by GerevSec** [comment]
Hey finswimmer,

Thanks for the info, I looked into it and you are of course correct.
I encountered another issue with installing the current preview on WSL2 ubuntu 20.04 with python 3.10 (the virtualenv package uses distutils_install["scripts"] which the PythonInfo object doesn't contain, so it's impossible to create the virtualenv.

Should I open a separate issue? I just bypassed it by using the master which has a newer virtualenv package.
Is the an expected date for the next alpha/beta release of poetry 1.2?
